aws-opentelemetry-distro/src/amazon/opentelemetry/distro/patches/_botocore_patches.py

```diff
@@ -178,7 +178,16 @@ def patch_extract_attributes(self, attributes: _AttributeMapT):
         if queue_url:
             attributes[AWS_SQS_QUEUE_URL] = queue_url
 
+    old_on_success = _SqsExtension.on_success
+
+    def patch_on_success(self, span: Span, result: _BotoResultT):
```

Comment:
is this a case we missed in the previous implementation?

Reply:
Yes this affects APIs like [SqsCreateQueue](https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/sqs/client/create_queue.html), where the `QueueUrl` is only available in the response body.